app/assets/stylesheets/custom.scss

```diff
@@ -387,3 +331,42 @@ h2, article h2 {
       }
    }
 }
+
+
+#tax-spent {
+        .content, #tax-spent-header, #tax-spent-footer {
+            display: flex;
+            flex-flow: row nowrap;
+            margin: 1em 0 0.2em;
+        }
+
+        .graph-green-bar {
+            border-bottom: 10px solid #28a197;
+        }
+
+        .graph-grey-bar {
+            border-top: 2px solid #b1b4b6;
+        }
+
+        dd {
+            flex-grow: 1;
+        }
+        dt {
+            flex-grow: 3;
+        }
+        dd {
+          text-align: right;
+        }
+        #tax-spent-header, #tax-spent-footer dt {
```

Comment:
My only comment about this would be, what about responsive font sizing on different screen sizes. I'll check, but is there not a font size mixin that includes responsiveness in v£.16 of AF?

Reply:
you could probably just use this class `class="heading-medium` from here to match bold 24, it will have the responsive font size built in.
https://govuk-elements.herokuapp.com/typography/#typography-headings